python/example_code/codecommit/push-in-parts.py

```diff
@@ -0,0 +1,239 @@
+# Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved. Licensed under the Amazon Software License (the "License"). 
+# You may not use this file except in compliance with the License. A copy of the License is located at 
+#    http://aws.amazon.com/asl/ 
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+
+# snippet-sourcedescription:[push-in-parts.py demonstrates how to migrate a Git repository in increments and repushes only those increments that did not succeed until the migration is complete.]
+# snippet-service:[codecommit]
+# snippet-keyword:[Python]
+# snippet-keyword:[AWS CodeCommit]
+# snippet-keyword:[Code Sample]
+# snippet-keyword:[none]
```

Comment:
You do not need the keyword none here. 
It is OK to delete this line completely, or you can add a keyword like Git.

Reply:
I have updated this with Git as requested.

Reply:
Thank you